recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -99,9 +99,14 @@ this.RecipeRunner = {
     }
   },
 
-  getFilterContext() {
+  getFilterContext(recipe) {
     return {
-      normandy: ClientEnvironment.getEnvironment(),
+      normandy: Object.assign(ClientEnvironment.getEnvironment(), {
```

Comment:
That's a good point. `getEnvironment` returns a new object so it doesn't matter in this case, but if it didn't this'd be a problem. I'll leave it for now.